src/data/ApolloClientManager/index.ts

```diff
@@ -828,14 +796,26 @@ export class ApolloClientManager {
     }
   };
 
-  completeCheckout = async (checkoutId: string) => {
+  completeCheckout = async (
+    checkoutId: string,
+    paymentData?: object,
+    redirectUrl?: string,
+    storeSource?: boolean
+  ) => {
```

Comment:
I have a feeling that it would be easier to deal with if it was an object extending this one:
https://github.com/mirumee/saleor-sdk/pull/21/files#diff-092c48c115782591404bfc4dd4ef628bR447-R451

Reply:
I have extracted it as a separate interface. The url with diff doesn't work - is the fixed version alright or should I need to change something else?